tasks/push-snapshot/tests/mocks.sh

```diff
@@ -28,14 +28,19 @@ function skopeo() {
   echo Mock skopeo called with: $*
   echo $* >> $(workspaces.data.path)/mock_skopeo.txt
 
-  if [[ "$*" != "inspect --no-tags --format {{.Digest}} docker://"* ]]
+  # Combined check for both command patterns
+  if [[ "$*" == "inspect --no-tags --format {{.Digest}} docker://"* ]] || [[ "$*" == "inspect docker://"* ]]
```

Comment:
This makes me think: Shouldn't we use `--no-tags` even for the source container image check? Won't it otherwise try to fetch all the tags of that container?

Reply:
+1

Reply:
Grateful you noticed that. Made the adjustment!

Reply:
OK, cool, thanks.